cloudscale/resource_cloudscale_server.go

```diff
@@ -354,6 +354,7 @@ func resourceServerCreate(d *schema.ResourceData, meta interface{}) error {
 	if attr, ok := d.GetOk("status"); ok {
 		originalStatus = attr.(string)
 	}
+	opts.Tags = CopyTags(d)
```

Comment:
I assume you found this issue yourself? It was not something that I reported?

Reply:
Yes, the tests were failing, because of this.